v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/Neo4jFlexTemplateOptions.java

```diff
@@ -0,0 +1,42 @@
+package com.google.cloud.teleport.v2.neo4j;
```

Comment:
I'm not sure why spotless doesn't highlight or fix this, but most (all?) the files in this PR seem to use incorrect indentation. E.g. blocks should have a 2-space indent, not 4-space.
See https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation

Reply:
I've run spotless again.